src/plugins/console/public/application/components/network_request_status_bar/network_request_status_bar.tsx

```diff
@@ -66,7 +68,8 @@ export const NetworkRequestStatusBar: FunctionComponent<Props> = ({
         </EuiBadge>
       </EuiFlexItem>
     );
-  } else if (requestResult) {
+    // If multiple requests are sent, we do not need to render status badges since they are displayed in the editor output panel
+  } else if (requestResult && !isMultipleRequest) {
```

Comment:
Let's revert this change, so we can continue to render the "time elapsed" information.

![image](https://user-images.githubusercontent.com/1238659/169880585-c48d14c8-fa73-4363-b47b-cca69b95a515.png)

This does introduce friction between the status code reported here and those reported in the results pane if they aren't all the same value. I think the status code in the header should show the first error encountered in the results pane. So if everything returns 200 except one request returns a 404, then I think the status code here should be 404. I think this is beyond the scope of this PR so after we merge this PR, can you create a followup issue to implement this change?

Reply:
Thanks for reverting this change. I created https://github.com/elastic/kibana/issues/132939 to track future enhancements about this behavior.